com.expidusos.file_manager.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.expidusos.file_manager
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+command: file_manager
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
if it supports wayland then it is `fallback-x11`

Reply:
Done